pypesto/predict/task.py

```diff
@@ -0,0 +1,46 @@
+import logging
+from typing import Sequence, Tuple
+
+from ..engine import Task
+
+logger = logging.getLogger(__name__)
+
+
+class PredictorTask(Task):
+    """Perform a single prediction with `pypesto.engine.Task`.
```

Comment:
```suggestion
    """Perform a single prediction with :class:`pypesto.engine.Task`.
```
will probably allow the link to be rendered correctly, same below.